apps/web/playwright/manage-booking-questions.e2e.ts

```diff
@@ -832,8 +832,6 @@ test.describe("Text area min and max characters text", () => {
       // wait until the button with data-testid="time" is visible
       await page.locator('[data-testid="time"]').isVisible();
 
-      await page.getByTestId("incrementMonth").click();
-
```

Comment:
let's keep these changes

Reply:
Will make them in another PR.

Reply:
but then E2E test will not pass and we'll not be able to merge this PR

Reply:
Right. Kept the changes.